gtime/model_selection/cross_validation.py

```diff
@@ -0,0 +1,99 @@
+import pandas as pd
+
+
+def time_series_split(time_series: pd.DataFrame, n_splits=4, split_on='index'):
+   """
+   Split the input DataFrame into n_splits. If the data is not a timeries then the split
```

Comment:
In general, it is not super clear the description

Reply:
Done